go/gosec/hardcoded_credentials/hardcoded_credentials.yaml

```diff
@@ -0,0 +1,16 @@
+rules:
+  - id: hardcoded-credentials
+    message: Potential hardcoded credentials
+    severity: WARNING
+    languages: [go]
+    patterns:
+      # len([cred_name == vars['$VALUE'] for cred_name in ['"secret"', 'passwd','pass','password','pwd', 'secret', 'token']]) > 1
+      - pattern-either:
+          - pattern: $VAR = "..."
+          - pattern: $VAR := "..."
+          - pattern: var $VAR = "..."
+          - pattern: const $VAR = "..."
+      - pattern-where-python: |
```

Comment:
This isn't a big deal for right now, but anecdotally we know that `token` and `secret` are prone to high FP. This was the case with the Bandit check which did the same thing: https://bento.dev/blog/2019/three-things-your-linter-shouldnt-tell-you/

Reply:
Yeah ideally, i want to add the entropy check to the literal so as to reduce the FP rate.